test/helpers.js

```diff
@@ -64,6 +66,24 @@ module.exports.getSwaggerApi = function (callback) {
   }
 };
 
+module.exports.getSwaggerApiRelativeRefs = function (callback) {
```

Comment:
Yea, GitHub hides the diff of that file by default, you have to explicitly load it 🙂 
Also, just wanna let you know, that diff of that file looks way more complicated on GitHub than it actually is. It's really just copying all of the tests, indenting them one more level and changing them to use `swagger-relative-refs`.